ABOUT.md

```diff
@@ -1,9 +1,106 @@
+The Shogun Machine learning toolbox offers a wide range of efficient and unified Machine Learning methods.
 
-One of Shogun's most exciting features is that you can use the toolbox through a *unified* interface from C++, Python, Octave, R, Java, Lua, C#, Ruby, etc. This not just means that we are independent of trends in computing languages, but it also lets you use Shogun as a vehicle to expose your algorithm to multiple communities. We use [SWIG](http://www.swig.org/) to enable *bidirectional* communication between C++ and target languages. See our [examples](http://shogun.ml/cookbook/latest/) in all target languages. Shogun runs under Linux/Unix, MacOS, and Windows. 
 
-Originally focussing on large-scale kernel methods and bioinformatics (for a list of scientific papers mentioning Shogun, see [here](http://scholar.google.com/scholar?hl=en&q=shogun+toolbox&btnG=&as_sdt=1%2C33&as_sdtp=)), the toolbox saw massive extensions to other fields in recent years. It now offers features that span the whole space of Machine Learning methods, including many classical methods in classification, regression, dimensionality reduction, clustering, but also more advanced algorithm classes such as metric, multi-task, structured output, and online learning, as well as feature hashing, ensemble methods, and optimization, just to name a few. Shogun in addition contains a number of exclusive state-of-the art algorithms such as a wealth of efficient SVM implementations, Multiple Kernel Learning, kernel hypothesis testing, Krylov methods, etc. All algorithms are supported by a collection of general purpose methods for evaluation, parameter tuning, preprocessing, serialisation & I/O, etc; the resulting combinatorial possibilities are huge. See our [showroom](http://shogun.ml/notebook/latest/).
+#### Shogun is accessible
 
-The wealth of ML open-source software allows us to offer bindings to other sophisticated libraries including: [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), [SVMLight](http://svmlight.joachims.org/), [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), [VowpalWabbit](http://www.hunch.net/~vw/), [Tapkee](http://tapkee.lisitsyn.me/), [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. See our [list of integrated external libraries](http://www.shogun-toolbox.org/page/about/contributions).
+* Supports *many languages* (Python, Octave, R, Java/Scala, Lua, C#, Ruby, etc) and *platforms* (Linux/Unix, MacOS and Windows), easy integration with other ML libraries
+
+#### Shogun is state-of-the-art
+
+* *efficient and competitive* implementation (from standard to cutting edge algorithms), modern software architecture in C++
```

Comment:
competitive because I was made to believe that Shogun's doing well enough on the benchmarking ... if not, I'll drop that. I agree that some of this might sound a bit unrealistic, but "Shogun works well most of the time and at least isn't worse than scikit learn as far as we can tell" does not make for a good header for example. Please, please ideas for a better way to write about these things guys!